c7n/resources/rdscluster.py

```diff
@@ -54,27 +54,43 @@ class Meta(object):
 class Delete(BaseAction):
 
     schema = type_schema(
-        'delete', **{'skip-snapshot': {'type': 'boolean'}})
+        'delete', **{'skip-snapshot': {'type': 'boolean'},
+                     'delete-instances': {'type': 'boolean'}})
 
-    def process(self, resources):
-        self.skip = self.data.get('skip-snapshot', False)
+    def process(self, clusters):
+        skip = self.data.get('skip-snapshot', False)
+        delete_instances = self.data.get('delete-instances', False)
```

Comment:
The cluster itself cannot be deleted if any of its instances are not in deleted/deleting state (an `InvalidDBClusterStateFault` exception is raised).
Agreed that we should default to *True*.
I'll change (and rebase).